website/docs/registry/faq.mdx

```diff
@@ -120,3 +120,9 @@ If your GPG key or Github account has been compromised, please [reach out to us
 ## Can I regain ownership of my artifact after losing access to my GitHub account?
 
 For support, please [reach out to us via email](/terraform/registry#getting-help).
+
```

Comment:
**Suggestion:** what if instead of adding a new question here, you instead add these sentences to the question ["How do I fix provider documentation issues?"](https://terraform-docs-common-bcjkdh2u9-hashicorp.vercel.app/terraform/registry/faq#how-do-i-fix-provider-documentation-issues) above. It feels like this question is related, and then all the information about rendering docs is grouped together at the top.

Reply:
This makes sense to me. Instead of it just being a sentence, can I add the h3 under the "How do I fix provider documentation issues?" question?

Reply:
I'd recommend double-checking the look of it in Vercel after making the change; but I think that makes sense!